packages/firebase_ml/lib/src/firebase_model_manager.dart

```diff
@@ -2,44 +2,46 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-
 part of firebase_ml;
 
-/// Manages the remote models.
+/// Before using a remote model,
+/// the user downloads it with [FirebaseModelManager].
 ///
-/// Before using a remote model, download it with [FirebaseModelManager].
-/// The model name is the key for a model. The model name should be consistent with the name of the model that has been uploaded to the Firebase console.
-/// https://firebase.google.com/docs/reference/android/com/google/firebase/ml/common/modeldownload/FirebaseModelManager
-class FirebaseModelManager{
-
-  static const MethodChannel _channel = MethodChannel('plugins.flutter.io/firebase_ml');
+/// The model name is the key for a model,
+/// and should be consistent with the name of the model
+/// that has been uploaded to the Firebase console.
+///
+/// https://firebase.google.com/docs/reference/android/com/google/
+/// firebase/ml/common/modeldownload/FirebaseModelManager
+class FirebaseModelManager {
+  static const MethodChannel _channel =
+      MethodChannel('plugins.flutter.io/firebase_ml');
 
   FirebaseModelManager._();
+
   /// Singleton of [FirebaseModelManager].
-  static FirebaseModelManager instance = FirebaseModelManager._();
+  static final FirebaseModelManager instance = FirebaseModelManager._();
 
   /// Initiates the download of remoteModel if the download hasn't begun.
-  Future<void> download( FirebaseRemoteModel model, FirebaseModelDownloadConditions conditions ) async {
-
+  Future<void> download(FirebaseRemoteModel model,
+      FirebaseModelDownloadConditions conditions) async {
     // Example call
     await _channel.invokeMethod(
```

Comment:
optional nit: for a simple `Future` like this, if you simply return the `Future` you don't need the `async` and `await` keyword

Reply:
I will keep it for now because the next step is modifying this function anyway